.evergreen/config.yml

```diff
@@ -2639,6 +2648,23 @@ buildvariants:
     - ".4.0"
     - ".3.6"
 
+- matrix_name: "test-macos-arm64"
+  matrix_spec:
+    platform:
+      - macos-1100-arm64
+    auth: "*"
+    ssl: "*"
```

Comment:
Let's use the `auth-ssl: "*"` axis instead. That way we can test auth/ssl noauth/nossl without needing the exclude_spec.

Reply:
Done.